src/clients/UBAClient/UBAClientWithRefresh.ts

```diff
@@ -12,7 +12,7 @@ export class UBAClientWithRefresh extends BaseUBAClient {
   constructor(
     readonly chainIdIndices: number[],
     readonly tokens: string[],
-    protected readonly hubPoolClient: HubPoolClient,
+    public readonly hubPoolClient: HubPoolClient,
```

Comment:
Curious about this change - it doesn't seem to be used elsewhere in this commit.

Reply:
I use it in the dataworker actually. I like enforcing that the dataworker is using the same clients that the uba client is using so it's events line up

Reply:
Yeah - if we are instantiating multiple HubPoolClient instances then we're doing something wrong. But the HubPoolClient is part of the CommonClients type, so doesn't the Dataworker already have a direct reference for the HubPoolClient?